frontend/src/App.tsx

```diff
@@ -60,8 +98,13 @@ function App() {
             Add
           </button>
         </div>
-
-        <TodoList todos={todos} />
+        {loading && <h3 className="h3-msg">LOADING...</h3>}
```

Comment:
Create a regular `const` variable and assign a value to it based on these conditions, and use that variable to render the content inside the component template.